app/models/manager_refresh/inventory_collection_default/infra_manager.rb

```diff
@@ -122,55 +208,81 @@ def ems_clusters(extra_attributes = {})
     def storages(extra_attributes = {})
       attributes = {
         :model_class => ::Storage,
-        :association => :storages,
+        # TODO: change :manager_ref => [:location],
+        :association => :storages
```

Comment:
I checked whether we can rely on `location` and noticed that in ovirt we have `ovirt-image-repository` storage where location is `nil`. There are code paths where we get storage info from disk and we do not have the location there. We would need to make additional call to fetch the location. Do we cross link ovirt storages as well?